diff --git a/node_modules/three/examples/jsm/loaders/FBXLoader.js b/node_modules/three/examples/jsm/loaders/FBXLoader.js
index 49ff2c7..c453a48 100644
--- a/node_modules/three/examples/jsm/loaders/FBXLoader.js
+++ b/node_modules/three/examples/jsm/loaders/FBXLoader.js
@@ -172,6 +172,8 @@ class FBXLoader extends Loader {
 
 		const textureLoader = new TextureLoader( this.manager ).setPath( this.resourcePath || path ).setCrossOrigin( this.crossOrigin );
 
+		this.fbxTree = fbxTree;
+
 		return new FBXTreeParser( textureLoader, this.manager ).parse( fbxTree );
 
 	}
